managed/data/iatemplates/agent_down.yml

```diff
@@ -8,6 +8,5 @@ templates:
     severity: critical
     annotations:
       description: |-
-        PMM agent on node '{{ $labels.node_id }}' cannot be reached. Host may be down.
-      summary: PMM agent on node '{{ $labels.node_id }}' cannot be reached. Host may be down.
-
+        PMM agent on node '{{ $labels.node_name }}', node ID '{{ $labels.node_id }}', cannot be reached. Host may be down.
```

Comment:
@cezmunsta Could you please check the description and the summary and let us know if they need more love?

Reply:
It wasn't so much about the description and summary, since the user can edit those, but that they could not add the information as it is not present. So long as the user isn't forced to try to manually determine which instance to look at, it seems fine